doozer/doozerlib/cli/scan_osh.py

```diff
@@ -90,8 +350,14 @@ def trigger_scans(self, nvrs: list):
                     self.runtime.logger.error("Invalid RHEL version")
                     return
```

Comment:
let's raise the error here?

Reply:
fixed in https://github.com/openshift-eng/art-tools/pull/141/commits/e4031289cbb0f229b3524b08cc04909ef204bc17